packages/back-end/src/app.ts

```diff
@@ -508,6 +508,7 @@ app.post(
 );
 app.post("/feature/:id/discard", featuresController.postFeatureDiscard);
 app.post("/feature/:id/publish", featuresController.postFeaturePublish);
+app.put("/feature/:id/archive", featuresController.archiveFeatureById);
```

Comment:
I would probably use `post` as the method instead of `put` to be consistent with the other actions.  Also, we typically name the controller functions starting with the method (e.g. `postFeatureArchive`).

Reply:
Changed in recent commit